setup.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+with open('CHANGELOG.md') as changelog_file:
+    changelog = changelog_file.read().replace('# Changelog', '')
```

Comment:
Yeah, i think I'm gonna change these markdown files to rst, so it looks better on PyPI.